frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -31,10 +31,14 @@ const Auth: React.FC = () => {
     const getScreen = (screen: string): React.ReactNode => {
         switch (screen) {
             case AppRoute.SIGN_IN: {
-                return <SignInForm onSubmit={handleSignInSubmit} />;
+                return (
+                    <SignInForm isTalent={true} onSubmit={handleSignInSubmit} />
```

Comment:
```suggestion
                    <SignInForm onSubmit={handleSignInSubmit} />
```
I think when we go to the signin form, we don't know who our client is